docs/data/material/components/tabs/NavTabs.tsx

```diff
@@ -34,6 +35,7 @@ function LinkTab(props: LinkTabProps) {
           event.preventDefault();
         }
       }}
+      aria-current={props.selected && "page"}
```

Comment:
```suggestion
      aria-current={props.selected ? "page" : undefined}
```

Reply:
Also, please add `aria-selected={undefined}` at the end to override the default behavior of the Tab.

```js
  <Tab
      component="a"
      onClick={(event) => {
        // Routing libraries handle this, you can remove the onClick handle when using them.
        if (samePageLinkNavigation(event)) {
          event.preventDefault();
        }
      }}
      aria-current={props.selected && "page"}
      {...props}
      aria-selected={undefined}
    />
```

Reply:
as I was checked, based on [w3 document](https://www.w3.org/TR/wai-aria-1.1/#aria-current) the default value of `aria-current` is false. 

> If the attribute is not present or its value is an empty string or undefined, the default value of false applies and the aria-current [state](https://www.w3.org/TR/wai-aria-1.1/#dfn-state) MUST NOT be exposed by user agents or assistive technologies.

Reply:
and for the second issue, we still need `aria-selected ` tag, because `aria-selected ` and `aria-current` represents different concepts and meanings here. 

based on [w3 document](https://www.w3.org/TR/wai-aria-1.1/#aria-current): 

> In some use cases for widgets that support aria-selected, current and selected can have different meanings and can both be used within the same set of elements.

Reply:
Thanks for the info!